webhooks/cluster.go

```diff
@@ -201,16 +201,17 @@ func (webhook *Cluster) validateTopology(ctx context.Context, old, new *clusterv
 
 	if old != nil { // On update
 		// Class could not be mutated.
-		if new.Spec.Topology.Class != old.Spec.Topology.Class {
-			allErrs = append(
-				allErrs,
-				field.Invalid(
-					field.NewPath("spec", "topology", "class"),
-					new.Spec.Topology.Class,
-					"class cannot be changed",
-				),
-			)
-		}
+		// FIXME(before merge): can be removed when https://github.com/kubernetes-sigs/cluster-api/pull/5644 has been merged.
```

Comment:
Will be rolled back **before** merge after https://github.com/kubernetes-sigs/cluster-api/pull/5644 has been merged